modules/test/playwright/tests/layout-set-prototype-web/siteTemplatesWithWebContent.spec.ts

```diff
@@ -56,6 +60,125 @@ const webContentName2: string = getRandomString();
 const webContentText1: string = getRandomString();
 const webContentText2: string = getRandomString();
 
+test('LPD-21445', async ({
+	apiHelpers,
+	applicationsMenuPage,
+	journalPage,
+	layoutSetPrototypePage,
+	page,
+	pageEditorPage,
+	pagesAdminPage,
+	productMenuPage,
+	sitesPage,
+	systemSettingsPage,
+	uiElementsPage,
+}) => {
+	const siteTemplateName: string = getRandomString();
+	const siteName1: string = getRandomString();
+	const siteName2: string = getRandomString();
+	const webContentName: string = getRandomString();
+	const text: string = getRandomString();
+	const secondPageNameOnSiteTemplate = getRandomString();
+
+	let site1Id: string | undefined;
+	let site2Id: string | undefined;
+
+	try {
+		await systemSettingsPage.disablePrivatePages();
+
+		await applicationsMenuPage.goToGlobalSite();
+		await productMenuPage.checkIfAdecuateProductMenu('Global');
+		await productMenuPage.openProductMenuIfClosed();
+		await productMenuPage.goToWebContent();
+		await journalPage.goToCreateArticle();
+		await journalPage.fillArticleData(webContentName, text);
+		await journalPage.publishArticle();
+
+		await createSiteTemplateWithContentPageAndAssetPublisher({
```

Comment:
in my local this fails because of issues with filling in the Web Content title, I have prepared a PR to fix this, maybe you can rebase it: https://github.com/brianchandotcom/liferay-portal/pull/153472